support/ebpf/types.h

```diff
@@ -386,7 +386,7 @@ typedef enum TraceOrigin {
 // MAX_FRAME_UNWINDS defines the maximum number of frames per
 // Trace we can unwind and respect the limit of eBPF instructions,
 // limit of tail calls and limit of stack size per eBPF program.
-#define MAX_FRAME_UNWINDS 128
+#define MAX_FRAME_UNWINDS 256
```

Comment:
does the Go code consume this value? as in does the Go code already attempt to read all frames?